packages/botonic-plugin-flow-builder/src/api.ts

```diff
@@ -248,11 +248,17 @@ export class FlowBuilderApi {
     if (this.flow.locales.find(flowLocale => flowLocale === locale)) {
       return locale
     }
-    const language = locale.split('-')[0]
-    if (this.flow.locales.find(flowLocale => flowLocale === language)) {
+
+    const language = locale?.split('-')[0]
+
+    if (
+      language &&
+      this.flow.locales.find(flowLocale => flowLocale === language)
+    ) {
       console.log(`locale: ${locale} has been resolved as ${language}`)
       return language
     }
```

Comment:
I think we can do this more robust in terms of resolving the locale, also I would find a way to make this more readable and not "comparing" `flowLocale` to `language`

Reply:
Yes, in the next PRs I will add logic to use a defaultLocale and refactor this part.